### PR TITLE
TestShowPartitions cleanup

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java [hunk 1]
```diff
@@ -71,7 +70,6 @@ private static TableDefinition partitionedTableDefinition()
 
     @Test(groups = {BASIC_SQL})
     public void testShowPartitionsFromHiveTable()
-            throws SQLException
```
Comment (kokosing) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java]: Can you remove all such things in product-tests

File: presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java [hunk 2]
```diff
@@ -56,17 +56,14 @@ public Requirement getRequirements(Configuration configuration)
...
     {
-        StringBuilder createTableDdl = new StringBuilder();
-        createTableDdl.append("CREATE EXTERNAL TABLE %NAME%(");
-        createTableDdl.append("   col INT");
-        createTableDdl.append(") ");
-        createTableDdl.append("PARTITIONED BY (part_col INT) ");
-        createTableDdl.append(" STORED AS ORC");
+        String createTableDdl = "CREATE EXTERNAL TABLE %NAME%(col INT) " +
+                "PARTITIONED BY (part_col INT) " +
+                "STORED AS ORC";
```
Comment (kokosing) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java]: You can also fixup ./src/main/java/com/facebook/presto/tests/hive/TestTablePartitioningSelect.java, it uses `StringBuilder` in similar way

File: presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java (hunk 1 above)
Comment (findepi) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java]: i'd rather not do this this time

File: presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java (hunk 2 above)
Comment (findepi) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java]: separate PR

**Pull request merged**